Please set email account credentials as environment variables for email to work.
 * Serving Flask app "server" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: off
 * Running on http://127.0.0.1:6080/ (Press CTRL+C to quit)
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[31m[1mDELETE /admin/user/remove/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[31m[1mDELETE /admin/user/remove/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[31m[1mDELETE /admin/user/remove/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mDELETE /admin/user/remove/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[31m[1mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:50] "[31m[1mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mDELETE /admin/user/remove/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mDELETE /admin/user/remove/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mDELETE /admin/user/remove/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /admin/user/remove/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /admin/userpermission/change/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/register/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/register/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/register/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/register/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/register/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/register/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/login/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/login/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/login/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /auth/login/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:51,691] ERROR in app: Exception on /auth/logout/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 53, in auth_logout_server
    token = str(request.json.get('token'))
AttributeError: 'str' object has no attribute 'get'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'AttributeError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:51] "[35m[1mPOST /auth/logout/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:51,725] ERROR in app: Exception on /auth/logout/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 53, in auth_logout_server
    token = str(request.json.get('token'))
AttributeError: 'str' object has no attribute 'get'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'AttributeError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:51] "[35m[1mPOST /auth/logout/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:51,758] ERROR in app: Exception on /auth/logout/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 53, in auth_logout_server
    token = str(request.json.get('token'))
AttributeError: 'str' object has no attribute 'get'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'AttributeError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:51] "[35m[1mPOST /auth/logout/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:51,790] ERROR in app: Exception on /auth/logout/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 53, in auth_logout_server
    token = str(request.json.get('token'))
AttributeError: 'str' object has no attribute 'get'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'AttributeError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:51] "[35m[1mPOST /auth/logout/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:51,830] ERROR in app: Exception on /channel/join/v2 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 95, in channel_join_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:51] "[35m[1mPOST /channel/join/v2 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /channel/join/v2 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[31m[1mPOST /channel/join/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:51] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[31m[1mPOST /channel/join/v2 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[31m[1mPOST /channel/join/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mGET /channel/details/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjY3LCJsb2dpbl90aW1lIjoxNjE5MTg2NTEyLCJzYWx0IjoiZWY0M2VhNDctYTEyZC00ZDNjLTgyN2QtNjRiMDVlZTUxMTQwIn0.5uQdg6FxAvqLEWXZ2DHQLU_6ziVUizbncEvmZm4J7LQ&channel_id=70 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,160] ERROR in app: Exception on /channel/addowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 106, in channel_addowner_server
    u_id = int(request.json.get('u_id'))
ValueError: invalid literal for int() with base 10: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjcyLCJsb2dpbl90aW1lIjoxNjE5MTg2NTEyLCJzYWx0IjoiYzMxMTNmYTUtM2ZkOS00NTY3LTljOGQtNDQxNWQxOGRkMTczIn0.rjPvFwiXM-RUjOA42bIcASfj6MTJwvWZW3sXiVpggzs'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/addowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,228] ERROR in app: Exception on /channel/addowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 106, in channel_addowner_server
    u_id = int(request.json.get('u_id'))
ValueError: invalid literal for int() with base 10: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzZXNzaW9uSWQiOjI0MzM0M30.VSx9GAW6jmgmatHb-ebckEa5Yb1_5j4hbTf2i9vKxUk'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/addowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,280] ERROR in app: Exception on /channel/addowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 105, in channel_addowner_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_channel'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/addowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,332] ERROR in app: Exception on /channel/addowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 105, in channel_addowner_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_channel'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/addowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[31m[1mPOST /channel/addowner/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/addowner/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/addowner/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mGET /channel/details/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjkxLCJsb2dpbl90aW1lIjoxNjE5MTg2NTEyLCJzYWx0IjoiYjNlZWFmNDYtYTcyNS00MzgxLTgyMzktN2Y4ZGI1MmZmNzI2In0.HGFPAdbSCie4caQ0SiDpqHEzsqI6T98bdL_OzaAe1yY&channel_id=93 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/addowner/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/removeowner/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/addowner/v1 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,574] ERROR in app: Exception on /channel/removeowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 117, in channel_removeowner_server
    u_id = int(request.json.get('u_id'))
ValueError: invalid literal for int() with base 10: 'invalid_user'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/removeowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/addowner/v1 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,643] ERROR in app: Exception on /channel/removeowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 116, in channel_removeowner_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_channel'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/removeowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,691] ERROR in app: Exception on /channel/removeowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 116, in channel_removeowner_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_channel'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/removeowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,742] ERROR in app: Exception on /channel/removeowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 116, in channel_removeowner_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_channel'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/removeowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,793] ERROR in app: Exception on /channel/removeowner/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 116, in channel_removeowner_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_channel'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/removeowner/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channel/leave/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mGET /channels/list/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjExMSwibG9naW5fdGltZSI6MTYxOTE4NjUxMiwic2FsdCI6ImM1OTFlMWUyLWM4Y2UtNDc5NS05MjQ0LTZlYThkYjdmNGNkNCJ9.xHbW1Gy3swaQcu6wZi3KMd7Z-ooE8ZSlAvZNHyLa0mI HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:52,864] ERROR in app: Exception on /channel/leave/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 127, in channel_leave_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_channel_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:52] "[35m[1mPOST /channel/leave/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[31m[1mPOST /channel/leave/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:52] "[37mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjExOSwibG9naW5fdGltZSI6MTYxOTE4NjUxMiwic2FsdCI6IjgwNDAwNjU5LTc0NGEtNDYyYi1iZWE1LWM1NThhYzE2OGVlMiJ9.gapS-xgOaLvgFM9tIBN92gT6qtAP0cZY6BsicATdxiU&channel_id=120&start=0 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjEyMiwibG9naW5fdGltZSI6MTYxOTE4NjUxMywic2FsdCI6IjBhNWUwMmI5LTM2YzUtNDNiYi04Mzg2LTI4MDkwZDg4NzQ3OCJ9.4yEnXIBvt1J7lcFHxHTg2ldRg_OR_DLJq_KrSg26Xgo&channel_id=50&start=0 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjEyMywibG9naW5fdGltZSI6MTYxOTE4NjUxMywic2FsdCI6IjIxNTdhZWNjLTBiMjItNDkyYi1iODg1LWE5NTk3MTc5MWUxZSJ9.1EDKDSV9Quf82QbbsshyGaQIb79Q6YK-5CfpCeUjmY0&channel_id=124&start=1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjEyNiwibG9naW5fdGltZSI6MTYxOTE4NjUxMywic2FsdCI6IjFlMTEwMzhkLTRiOTEtNGFjOC1hYmZjLWMwMTUyY2EwOTc4ZiJ9.zMbGrvgaLxIY_c1sr_gKvdPsMfXbTlWSSJx43fIfaq8&channel_id=127&start=0 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjEyOCwibG9naW5fdGltZSI6MTYxOTE4NjUxMywic2FsdCI6IjU1YWY0MjJmLWNiYzAtNGFkZi04ODg0LWZiYTJlZDgwYTExMCJ9.zOAmFlDELSETv3aBwu8YVYerZSH5bC-HQ9gmW8hZ8ng&channel_id=129&start=0 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /channels/create/v2 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /channels/create/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mGET /channels/list/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjEzOCwibG9naW5fdGltZSI6MTYxOTE4NjUxMywic2FsdCI6IjY2MmU0YmNhLTk2ODMtNDhmMS04OTQ4LWEyMDlkZjgwNjVlZCJ9.pnvFmCf5r0ebAXJLm4Wk9fZHZbsRgwONtN_SlyFXy6w HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mGET /channels/list/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzZXNzaW9uSWQiOjk5OX0.erYxCQ9czIu14IV0KRb7E4AecDGf1Ta1GmmRQpejfM4 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mGET /channels/listall/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjE0MiwibG9naW5fdGltZSI6MTYxOTE4NjUxMywic2FsdCI6IjRmMWExM2Q4LTIwOGMtNGM3MS1iYjBlLTUwYTg3ZDUyODMzZSJ9.a14j71R8TbAYtbWf9VF0nRgbgcZl_Gp_46QfBB7yR4o HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mGET /channels/listall/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzZXNzaW9uSWQiOjk5OX0.erYxCQ9czIu14IV0KRb7E4AecDGf1Ta1GmmRQpejfM4 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/invite/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mGET /dm/list/v1?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjE1MywibG9naW5fdGltZSI6MTYxOTE4NjUxMywic2FsdCI6IjY3OThlZjM5LTRlN2MtNDM5MS05OTZmLWFjZmUyYTQyYThhZSJ9.j6qpMQyaB3tKaYBDgrjW1T-krh0sNxsr1QzpHSXj6b4 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/invite/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/invite/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/invite/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mGET /dm/list/v1?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjE2NSwibG9naW5fdGltZSI6MTYxOTE4NjUxMywic2FsdCI6ImFlZTk5NTI2LTEzMzgtNDc1Ny1iMDBlLWZmYmUzOGE2NzNhNCJ9.yIXBXxg6f3uWBrc15biFfqxHc4KF1FpCxF_migPWs-E HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mGET /dm/list/v1?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzZXNzaW9uSWQiOjd9.1YEfeZ4MlQoTQ1j2eKs2VRXUZD8tjD7nfXjg0BvBM24 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
[2021-04-24 00:01:53,905] ERROR in app: Exception on /dm/leave/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 261, in dm_leave_server
    dm_id = int(request.json.get('dm_id'))
ValueError: invalid literal for int() with base 10: 'invalid_dm_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:53] "[35m[1mPOST /dm/leave/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:53] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
[2021-04-24 00:01:54,122] ERROR in app: Exception on /dm/remove/v1 [DELETE]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 240, in dm_remove_server
    dm_id = int(request.json.get('dm_id'))
ValueError: invalid literal for int() with base 10: 'invalid_dm_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:54] "[35m[1mDELETE /dm/remove/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mGET /dm/messages/v1?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjIwMSwibG9naW5fdGltZSI6MTYxOTE4NjUxNCwic2FsdCI6IjEzMzc1MzlmLTY2ZWYtNDEyOS05NjM5LTBlOWQxYmMwMGYwYyJ9.Wyd1672qrU6WtVVvzBgim3V9kxz-MDUzDxAup67tjpg&dm_id=999&start=0 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mGET /dm/details/v1?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjIxOSwibG9naW5fdGltZSI6MTYxOTE4NjUxNCwic2FsdCI6ImI5YmYyNTBjLWExMjQtNGZiMy04MGYwLTdlMjY0YWYyMzRlOCJ9.sZe4yyXh1jk4P5YMr9xx5oIt74aZvrYv4xctyKy_l3Y&dm_id=220 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mGET /dm/details/v1?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjIyMSwibG9naW5fdGltZSI6MTYxOTE4NjUxNCwic2FsdCI6IjY1MTFiYTIzLWYzZDYtNDg0Zi05NzUwLTAwZjMxZmRhZGIwMiJ9.9vdFB8mudKsbeeb62nq3xDTI-jqopui7m0XITTFffhI&dm_id=223 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[33mGET /echo?data=hello HTTP/1.1[0m" 404 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /message/send/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /message/send/v2 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:54,894] ERROR in app: Exception on /message/send/v2 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 166, in message_send_server
    channel_id = int(request.json.get('channel_id'))
ValueError: invalid literal for int() with base 10: 'invalid_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:54] "[35m[1mPOST /message/send/v2 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[31m[1mPOST /message/send/v2 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:54] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjIzNiwibG9naW5fdGltZSI6MTYxOTE4NjUxNCwic2FsdCI6IjJmOTQ0NGRiLTM0ODEtNGE1NS1hNzY4LWQ4NzdiNGE2Zjc4ZCJ9.DeUAUdUKW2q7i4nUHgkIoDRdgOsemX25p1kG67IvHVA&channel_id=237&start=0 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPUT /message/edit/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[31m[1mPUT /message/edit/v2 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[31m[1mPUT /message/edit/v2 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:55,238] ERROR in app: Exception on /message/edit/v2 [PUT]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 177, in message_edit_server
    message_id = int(request.json.get('message_id'))
ValueError: invalid literal for int() with base 10: 'invalid_mID'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:55] "[35m[1mPUT /message/edit/v2 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[31m[1mPUT /message/edit/v2 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPUT /message/edit/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjI1NSwibG9naW5fdGltZSI6MTYxOTE4NjUxNSwic2FsdCI6ImE2NjMzZmUwLTJjMTItNDhkNC04ZjgxLTU4YzRmNmNiYzk3NyJ9.jDZOF875wYiYrpacZ2BFllqrtGUpDpODGwqHjjDckjM&channel_id=256&start=0 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPUT /message/edit/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjI1OSwibG9naW5fdGltZSI6MTYxOTE4NjUxNSwic2FsdCI6ImM5ZjQyMDI0LTdhMTUtNGRjMy05YzExLWUxZGNmZDFjNGY3MSJ9.oOWUsRQfmHGlpiuWM9ITDRrED08PD1PDJYCDo5XHQzQ&channel_id=260&start=0 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[31m[1mDELETE /message/remove/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:55,535] ERROR in app: Exception on /message/remove/v1 [DELETE]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 188, in message_remove_server
    message_id = int(request.json.get('message_id'))
ValueError: invalid literal for int() with base 10: 'invalid_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:55] "[35m[1mDELETE /message/remove/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:55,584] ERROR in app: Exception on /message/remove/v1 [DELETE]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 188, in message_remove_server
    message_id = int(request.json.get('message_id'))
ValueError: invalid literal for int() with base 10: 'm_id'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:55] "[35m[1mDELETE /message/remove/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[31m[1mDELETE /message/remove/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /message/remove/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjI3MywibG9naW5fdGltZSI6MTYxOTE4NjUxNSwic2FsdCI6IjlmODYwMTNhLTUwNmUtNDg4Mi04ZWFlLTU5MWU3YjliMWRkZiJ9._yUKMjewTF787B-2NvmcULtbhyWU0cdPFVVCwvawyEs&channel_id=274&start=0 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /channel/join/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /message/remove/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mGET /channel/messages/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjI3NywibG9naW5fdGltZSI6MTYxOTE4NjUxNSwic2FsdCI6IjBmNjQ0N2Q1LTJjNTktNGU2Ny04YTE0LWVlZTEwYjllMDJkNiJ9.nSZEBg_3qhqzz1fjrZPrU2-QTcIqopNrnJWKhap5Vrg&channel_id=278&start=0 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /message/senddm/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[31m[1mPOST /message/senddm/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[31m[1mPOST /message/senddm/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
[2021-04-24 00:01:55,928] ERROR in app: Exception on /message/senddm/v1 [POST]
Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/server.py", line 282, in message_senddm_server
    dm_id = int(request.json.get('dm_id'))
ValueError: invalid literal for int() with base 10: 'invalid_dmID'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/import/kamen/3/cs1531/.local/lib/python3.7/site-packages/flask/app.py", line 1822, in handle_user_exception
    return handler(e)
  File "/tmp_amd/kamen/export/kamen/3/cs1531/automark/src/api/error.py", line 5, in default_handler
    response = err.get_response()
AttributeError: 'ValueError' object has no attribute 'get_response'
127.0.0.1 - - [24/Apr/2021 00:01:55] "[35m[1mPOST /message/senddm/v1 HTTP/1.1[0m" 500 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:55] "[31m[1mPOST /message/senddm/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/senddm/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mGET /dm/messages/v1?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjI5OCwibG9naW5fdGltZSI6MTYxOTE4NjUxNiwic2FsdCI6ImMyM2M4ZGE3LTI5YmUtNDFlMy04ODIzLTIyZDlkOTY1ZjdiMSJ9.G-hVBEDq0NjYy8tB9dcnoGyHIdOwOSWoVG9_C4kdho4&dm_id=299&start=0 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[31m[1mPOST /message/share/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/senddm/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[31m[1mPOST /message/share/v1 HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/share/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/share/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/senddm/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/share/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /dm/create/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/senddm/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/share/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[31m[1mPOST /dm/create/v1 HTTP/1.1[0m" 400 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /channels/create/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /message/send/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mGET /search/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJnaWQiOjMzNiwibG9naW5fdGltZSI6MTYxOTE4NjUxNiwic2FsdCI6IjllMjNiZjY3LTI5NzItNGVlZS05NzZmLWQ2MDIzNWQ3NjczZiJ9.gAmOa6wqW6vyGEC7LQngwc9MB2Rauusy-xUP7u5wbec&query_str=Hello HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[31m[1mGET /search/v2?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzZXNzaW9uSWQiOjd9.1YEfeZ4MlQoTQ1j2eKs2VRXUZD8tjD7nfXjg0BvBM24&query_str=Hello HTTP/1.1[0m" 403 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mDELETE /clear/v1 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
127.0.0.1 - - [24/Apr/2021 00:01:56] "[37mPOST /auth/register/v2 HTTP/1.1[0m" 200 -
